packages/contracts/tasks/helpers/Deployment.ts

```diff
@@ -377,6 +390,17 @@ export class Deployment {
     });
     await contract.deploymentTransaction()!.wait();
 
+    if (signer && (signer as AASigner).isAA) {
```

Comment:
I added this condition in the `deployContractWithLinkedLibraries` function. I have to modify some upper functions calling this one in order to send the signer all the way here. 

Let me comment where I modify these functions